site/package.json

```diff
@@ -37,7 +38,68 @@
     "remark-html": "^15.0.2",
     "typescript": "5.2.2"
   },
+  "eslintConfig": {
+    "root": true,
+    "extends": [
+      "next",
+      "next/core-web-vitals"
+    ],
+    "plugins": [
+      "@typescript-eslint"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "rules": {
+      "react/prop-types": [
+        2,
+        {
+          "skipUndeclared": true
+        }
+      ],
+      "react/jsx-handler-names": "off",
+      "quotes": "off",
+      "jsx-quotes": [
+        2,
+        "prefer-single"
+      ],
+      "comma-dangle": [
+        2,
+        "always-multiline"
+      ],
+      "object-curly-spacing": [
+        2,
+        "always"
+      ],
+      "react/display-name": 0,
+      "semi": [
+        "error",
+        "never"
+      ],
+      "no-underscore-dangle": 0,
+      "space-before-function-paren": 0,
+      "arrow-body-style": 0,
+      "no-use-before-define": 0,
+      "arrow-parens": 0,
+      "no-trailing-spaces": "error",
+      "@typescript-eslint/quotes": [
+        "error",
+        "single"
+      ],
+      "@typescript-eslint/indent": [
+        "error",
+        4
```

Comment:
Did you change this to 4? Can we use 2 for TS and 4 for Rust?

Reply:
Ah, yes. Looks like I forgot to revert some experimentation. I'll change that back to `2`. About Rust, we currently don't have a `.rustfmt.toml` (or indeed a `cargo fmt --check` step in the CI). I'll have a look at these and experiment with some settings.

Reply:
@calummoore So the default spacing in `rustfmt` is 4 spaces (which is what we want), and since we don't change any of the default values, we can probably eschew using an explicit `.rustfmt.toml` until we do. 

I noticed some (very few) discrepancies when running `cargo fmt` on this repository as well as the `Polylang` repository. I will add an extra step for `cargo fmt --check` (using the default settings) in the CI for both repositories. Thoughts?

@mateuszmlc